internal/agent/install.go

```diff
@@ -63,28 +63,34 @@ func displayInfo(agentConfig *Config) {
 }
 
 func ManualInstall(c string, options map[string]string, strictValidations bool) error {
-	dat, err := os.ReadFile(c)
-	if err != nil {
-		return err
-	}
+	if _, err := url.Parse(c); err == nil {
```

Comment:
This looks good, however can we isolate that in a function and have it covered by tests?